### ISSUE TITLE
Lack of accessibility controls to accommodate screen resolution

### ISSUE THREAD

**Issue by gnidan** [trufflesuite/ganache#1262]
## Expected Behavior

I would expect cmd-plus / cmd-minus (or other system equivalents) to be available keyboard shortcuts for making the fonts bigger / smaller, or just do the regular webkit zoom thing.

## Current Behavior

Nothing!

## Possible Solution

Does Electron allow this capability easily?

## Context

I was presenting Ganache to a live audience and they couldn't read the small text.

## Your Environment

Ganache v2.0.1 on Mac

**Comment by davidmurdoch** [comment]
For future reference: https://stackoverflow.com/a/40278246/160173
